packages/cli/src/ui/App.tsx

```diff
@@ -479,7 +479,10 @@ const App = ({ config, settings, startupWarnings = [], version }: AppProps) => {
       if (Object.keys(mcpServers || {}).length > 0) {
         handleSlashCommand(newValue ? '/mcp desc' : '/mcp nodesc');
       }
-    } else if (key.ctrl && (input === 'c' || input === 'C')) {
+    } else if (
+      (key.ctrl && (input === 'c' || input === 'C')) ||
+      input === '[99;5u'
```

Comment:
perhaps we should grab this and other kitty(?) related constants and add them to a file of constants along with brief documentation of what each of them means